### PR TITLE
Adds troubleshooting to sidebar.

### LINKED ISSUE #153: Add a Troubleshooting page for common issues/suggestion
A [comment in another issue](https://github.com/web3-storage/docs/issues/127#issuecomment-894353098) raised the idea of having a troubleshooting page to help users figure out why stuff is going wrong.

It'd be nice to have a braindump in here for common problems users are running into with Web3.Storage.

## Problems

- We'd like all users to run Webpack 5, but some devs are forced into using Webpack 4 because of other dependencies they've got (React, for example, only runs Webpack 4).

### PR DESCRIPTION
**Pull request by johnnymatthews**
Closes #153 by creating a troubleshooting page.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-tos/troubleshooting.md`:
- Consider a shorter alternative to avoid wordiness.
  ```
  ...frameworks like React require Webpack 4 in order to run properly. To get aro...
                                             ^^^^^^^^^^^
  ```
- With the double genitive (double possessive), use the possessive pronoun “yours”.
  ```
  ... add the following import to the top of your .js scripts to import the pre-w...
                                             ^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-tos/troubleshooting.md`:
- Consider a shorter alternative to avoid wordiness.
  ```
  ...frameworks like React require Webpack 4 in order to run properly. To get aro...
                                             ^^^^^^^^^^^
  ```
- With the double genitive (double possessive), use the possessive pronoun “yours”.
  ```
  ... add the following import to the top of your .js scripts to import the pre-w...
                                             ^^^^
  ```

File: docs/how-tos/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+description: A collection of common issues for new developers, and solutions to those problems.
+---
+
+# Troubleshooting
+
+This page contains a collection of common issues for new developers, and solutions to those problems.
+
+## I need to use Webpack 4
+
+We recommend using Webpack 5 with your peojcts. However, some frameworks like React require Webpack 4 in order to run properly. To get around this issue, add the following import to the top of your `.js` scripts to import the pre-webpack-bundled version of Web3.Storage:
```
Comment (yusefnapora) [inline docs/how-tos/troubleshooting.md:12]:
```suggestion
We recommend using Webpack 5 with your projects. However, some popular tools like create-react-app require Webpack 4 in order to run properly. To get around this issue, add the following import to the top of your `.js` scripts to import the pre-webpack-bundled version of Web3.Storage:
```

I think you can use React with webpack 5, but tons of people use create-react-app, which is still on v4.

**Review by yusefnapora — APPROVED** [review]
works for me :)

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-tos/troubleshooting.md`:
- Consider a shorter alternative to avoid wordiness.
  ```
  ...like create-react-app require Webpack 4 in order to run properly. To get aro...
                                             ^^^^^^^^^^^
  ```
- With the double genitive (double possessive), use the possessive pronoun “yours”.
  ```
  ... add the following import to the top of your .js scripts to import the pre-w...
                                             ^^^^
  ```

**Pull request merged**